### DISCUSSION TITLE
Cache specific message

### DISCUSSION THREAD

**Discussion by Matze1211** [discordjs/discord.js#5677] [Q&A] [answered]
Is there a way to cache a specific message once the client connects so my bot can listen to reactions to said message?

**Comment by kyranet** [comment, marked as answer]
You can do this by enabling the [`MESSAGE`](https://discord.js.org/#/docs/main/stable/typedef/PartialType) partial in your bot's client option, although that'll also make other events such as [messageUpdate](https://discord.js.org/#/docs/main/stable/class/Client?scrollTo=e-messageUpdate) and [messageDelete](https://discord.js.org/#/docs/main/stable/class/Client?scrollTo=e-messageDelete) be fired for any other message.

**Reply by Matze1211** [reply]
It seems I had to activate both MESSAGE and GUILD_MEMBER, but that worked, thanks!
